app/app.global.css

```diff
@@ -29,3 +29,21 @@ a:hover {
   text-decoration: none;
   cursor: pointer;
 }
+
+.marker {
+  background-image: url('../resources/mapbox-icon.png');
```

Comment:
FWIW I assume we will change this, this image is straight from a tutorial

Reply:
ideally this would be one of the images spotted there

Reply:
but this is outside the scope of this PR , just FYI

Reply:
:+1: